flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/rules/logical/FlinkFilterJoinRule.java

```diff
@@ -449,6 +455,33 @@ protected FlinkFilterIntoJoinRule(FlinkFilterIntoJoinRule.Config config) {
             super(config);
         }
 
+        @Override
+        public boolean matches(RelOptRuleCall call) {
+            Join join = call.rel(1);
+            return !isEventTimeTemporalJoin(join.getCondition()) && super.matches(call);
+        }
```

Comment:
Can this `isEventTimeTemporalJoin` judgment use `TemporalJoinUtil.containsTemporalJoinCondition` or extracting `is event time temporal join judgment`  into this util class(` TemporalJoinUtil` ).

Reply:
> 

I got it.  Thanks for you reply.